Sources/Basic/OutputByteStream.swift

```diff
@@ -55,27 +60,89 @@ public class OutputByteStream: TextOutputStream {
         return buffer.count
     }
 
-    /// The contents of the output stream.
-    ///
-    /// This method implicitly flushes the stream.
-    public final var bytes: ByteString {
-        flush()
-        return ByteString(self.buffer)
+    /// Currently available buffer size.
+    private var availableBufferSize: Int {
+        // FIXME: max() is not really needed below because buffer's capacity should
+        // never exceed the defined buffer size. But it is defensive until we move to
+        // a buffer whose memory OutputByteStream manages.
+        return max(buffer.capacity, OutputByteStream.bufferSize) - buffer.count
```

Comment:
I don't think we need `max` at all, I think we can always use buffer.capacity, right?